src/main/java/com/faforever/client/chat/UserFilterController.java

```diff
@@ -30,20 +33,24 @@
 public class UserFilterController implements Controller<Node> {
 
   private final I18n i18n;
+  private CountryFlagService flagService;
   public MenuButton gameStatusMenu;
   public GridPane filterUserRoot;
   public TextField clanFilterField;
   public TextField minRatingFilterField;
   public TextField maxRatingFilterField;
   public ToggleGroup gameStatusToggleGroup;
+  public ComboBox countryFilterField;
```

Comment:
A ComboBox of what. See how Intelij warns you. In order to achieve type safety please use the Type parameter:
```suggestion
  public ComboBox<String> countryFilterField;
```

Reply:
Ended up going with a TextField instead. The ComboBox was confusing as you could end up seeing values that didn't match what you were typing.